### PR TITLE
docs: add example for nodejs lintText api

### PR DESCRIPTION
**Pull request by siva-kannan3**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

https://github.com/eslint/eslint/issues/16666

#### What changes did you make? (Give an overview)

Added a basic example for eslint lintText API in eslint node docs.

#### Is there anything you'd like reviewers to focus on?

No

### CODE REVIEW

**Comment by siva-kannan3** [comment]
@mdjermanovic,
Since the pr https://github.com/eslint/eslint/pull/16665 was merged, can I mark this pr ready for review?

File: docs/src/integrate/nodejs-api.md [hunk 1]
```diff
@@ -24,48 +23,92 @@ Here's a simple example of using the `ESLint` class:
...
+            env: {
+                es2022: true,
+                node: true,
+            },
+        },
+    });
 
-  // 3. Modify the files with the fixed code.
-  await ESLint.outputFixes(results);
+    // 2. Lint files.
```
Comment (aladdin-add) [inline docs/src/integrate/nodejs-api.md:100]:
```suggestion
    // 2. Lint text.
```

**Comment by aladdin-add** [comment]
no worries, I've marked as "ready for review". :)

**Review by nzakas — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**